OpenRA.Mods.Common/Projectiles/InstantHit.cs

```diff
@@ -71,7 +71,11 @@ public void Tick(World world)
 
 			if (!doneDamage)
 			{
-				args.Weapon.Impact(Target.FromPos(target), args.SourceActor, args.DamageModifiers);
+				if (info.Inaccuracy.Length > 0)
```

Comment:
I might be missing something obvious, but your `args.PassiveTarget` isn't going to change between the constructor and here, so there shouldn't be any difference between applying the inaccuracy in the constructor vs here.